app/models/like.rb

```diff
@@ -0,0 +1,13 @@
+class Like < ApplicationRecord
+  belongs_to :user, foreign_key: 'author_id'
```

Comment:
- [ ] nice job so far. what the previous reviewer meant was that the belongs_to side between Like and User is formated the same as in the Post class as such

```
belongs_to :author, class_name: 'User'
belongs_to :post

```
please do same for the comment model